.gitignore

```diff
@@ -10,3 +10,6 @@ yarn.lock
 validator.js
 validator.min.js
 index.js
+
+# Ignore VSCode workspace config
+workspace.code-workspace
```

Comment:
I don't think vendor specific ignore list should be added to the lib. Not everybody is using VSCode

Reply:
I think that excluding vendor specific files would help to reduce the risk of codebase pollution. In fact `.DS_Store` files are ignored even if their are MacOS specific.